Dockerfile

```diff
@@ -17,3 +17,8 @@ RUN go install -v ./cmd/...
 FROM alpine:3.11
 RUN apk add --no-cache ca-certificates iptables
 COPY --from=build-env /go/bin/* /usr/local/bin/
+
+CMD ["/usr/local/bin/tailscaled", "--state=/var/lib/tailscale/tailscaled.state", "--socket=/var/run/tailscale/tailscaled.sock", "--port=41641"]
+
+
+# docker run --network=host  --cap-add=NET_ADMIN --device /dev/net/tun:/dev/net/tun myusuf3/tailscale
```

Comment:
We don't want `myusuf3` in there.

Also, we already added docs to this file earlier:

https://github.com/tailscale/tailscale/blob/master/Dockerfile#L5

The --state and --socket flags weren't required for me to make it work.